### DISCUSSION TITLE
Would there be Linux support?

### DISCUSSION THREAD

**Discussion by xendyex** [geode-sdk/sdk#190] [Geode Ideas]
I'm talking about either NATIVE linux support, or, an easy to follow tutorial on how to run it on Proton (steam compatibility layer)

**Would there be LINUX support?**

**Comment by Fleeym** [comment]
Can't have linux support if the game isn't native on linux. 
You can run it with Proton, just add `WINEDLLOVERRIDES="XInput9_1_0.dll=n,b" %command%` in your launch options. Also make sure to run with proton 7.

To install it just do it [manually](https://docs.geode-sdk.org/installation#installing-geode-manually)

**Reply by xendyex** [reply]
Is there any chance I could use Proton 8.0.2? It's what I usually use.

**Reply by Fleeym** [reply]
you can try it, I had no luck with proton 8 and GD

**Reply by xendyex** [reply]
I tried, no work. I backed-up GD though, so all is fine. I will try with P7 later in the week; busy today! Thanks, though.

**Reply by Malo1301** [reply]
When 2.11 was still around, I used Geode with Proton 8.0-4. To install it you need to launch the installer with wineprefix ~/.steam/steam/steamapps/compatdata/322170/pfx (I used Lutris to set a wineprefix only for the installer) (Path can vary between distros, this path is the one I have on Ubuntu but it is not the same for every distros) and after the installation is done, just add WINEDLLOVERRIDES="XInput9_1_0.dll=n,b" %command% to your launch options. Note that I used Proton 8.0-4 to run the Geode installer, you can do it if you use Lutris by copying your Proton installation folder to the Wine versions folder wich is in the Lutris folder, somewhere. I highly recommand using Lutris if you want to install mods like MegaHack or Geode on Linux.

**Reply by RuralAnemone** [reply]
to anyone looking for an answer in the future, this worked for me:

- install [`protontricks`](https://github.com/Matoking/protontricks)

- find the app ID of the game you're trying to patch (afaik proton sandboxes steam games??)

- `protontricks-launch --appid APP_ID_GOES_HERE path/to/executable.exe`

**Reply by bp88dev** [reply]
The link's leading to a 404

**Reply by fluffykanade** [reply]
🥀
![image](https://github.com/user-attachments/assets/f7fb0e39-3535-4f7c-b781-62858a54e1da)

**Reply by RoootTheFox** [reply]
the easiest way to [install geode on linux](https://geode-sdk.org/install) is to just run
```bash
curl -o- 'https://geode-sdk.org/install/linux.sh' | bash
```
in a terminal
